main.go

```diff
@@ -169,6 +172,52 @@ func filterBucketContent(elems s3.ListObjectsOutput, csvSuffix, processedFlagSuf
 	return unprocessed, processed, orphanFlags
 }
 
+func filterOutNewerWindows(objects s3.ListObjectsOutput, newestWindowCountToIgnore int) (olderObjects s3.ListObjectsOutput) {
+	// Remove the <newestWindowCountToIgnore> most recent folders from the object listing in order to not clean them
+	windowedObject := make(map[int64][]types.Object)
+	var windows []int64
+
+	if newestWindowCountToIgnore == 0 {
+		return objects
+	}
```

Comment:
Also just added this, to avoid raising errors when we do not have the timestamp in the bucket, and do not want to use this feature.